suite-native/assets/src/components/Assets.tsx

```diff
@@ -39,6 +39,29 @@ export const Assets = () => {
         selectAssetsWithBalances(state, fiatCurrency.label),
     );
 
+    const assetsDataWithPercentage = useMemo(() => {
+        const fiatTotal = assetsData.reduce((sum, next) => sum + +next.fiatBalance, 0);
```

Comment:
If you need convert string to number, please use `Number(next.fiatBalance)` it's better for readability. 

Ideally you should use BigNumber.js to work with these values, otherwise numbers is loosing precision quite a lot, but in this case it's probably OK to if it's not totally precise.